src/client/js/otp/util/Itin.js

```diff
@@ -321,6 +334,46 @@ otp.util.Itin = {
         'agencyId': parts.shift(),
         'id': parts.join(':')
       }
-    }
+    },
+
+    getModeColor : function(mode) {
```

Comment:
This is probably from #3908, isn't it?

Reply:
Yes, I rebased onto the wrong branch. I've removed those commits.